kickstarts/partials/post/ruby_install.ks.erb

```diff
@@ -1,23 +1,9 @@
-curl -L https://github.com/postmodern/ruby-install/archive/v0.6.0.tar.gz | tar xvfzC - /tmp
-
-pushd /tmp/ruby-install*/
-make install
-popd
-
-# The ruby-install will result in about 350MiB of files in /usr/local/src.
-# These do not need to be in the image, so put them on a tmpfs.
-mkdir -p /usr/local/src
-mount -t tmpfs tmpfs /usr/local/src
-
 # Note, we hardcode the ruby patch level for now.
 # Once the --update option is available, we could ask
 # ruby-install to "learn" what new rubies are available
 # See: https://github.com/postmodern/ruby-install/issues/175
```

Comment:
Also in case this is just the installer - is there any way we could install Ruby from RPM from e.g. softwarecollections.org? My measurements indicated that this should reduce the build time of the application by about 7-10 minutes.